source/_components/device_tracker.owntracks_http.markdown

```diff
@@ -47,17 +47,6 @@ sensor:
         friendly_name: Pixel of Paulus
         unit_of_measurement: "%"
         value_template: '{{ states.device_tracker.paulus_pixel.attributes.battery|int }}'
-        icon_template: >-
-          {% set battery_level = states.device_tracker.paulus_pixel.attributes.battery|default(0)|int %}
-          {% set battery_round = (battery_level / 10) |int * 10 %}
-          {% if battery_round >= 100 %}
-            mdi:battery
-          {% elif battery_round > 0 %}
-            mdi:battery-{{ battery_round }}
-          {% else %}
-            mdi:battery-alert
-          {% endif %}
-        entity_id: 
-        - device_tracker.paulus_pixel
```

Comment:
Manual `entity_id` selection should no longer be necessary in most cases (including this one). So to keep the docs a bit simple, I removed it here.